### PR TITLE
releases: add TiDB 3.0.19 release notes

### PR DESCRIPTION
**Pull request by ti-srebot**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: releases/release-3.0.19.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Others
+
++ TiDB
+
+    - Fix the issue that alter enum/set type does not check constraint [#20045](https://github.com/pingcap/tidb/pull/20045)
+    - Fix the panic that tidb-server does not release table locks [#20021](https://github.com/pingcap/tidb/pull/20021)
+    - Fix the behavior of rewrite ScalarFunction IsTure . [#19901](https://github.com/pingcap/tidb/pull/19901)
+    - Support adjusting the concurrency on the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
```
Comment (youjiali1995) [inline releases/release-3.0.19.md:34]: lgtm

File: releases/release-3.0.19.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    - Set `sync-log` to `true` always [#8636](https://github.com/tikv/tikv/pull/8636)
+    - Fix the bug that TiKV panics when parsing responses with missing reason phrases [#8540](https://github.com/tikv/tikv/pull/8540)
+
++ PD
+
+    - Add an alert rule for restart [#2789](https://github.com/pingcap/pd/pull/2789)
```
Comment (nolouch) [inline releases/release-3.0.19.md:41]: move to `Improvements`

File: releases/release-3.0.19.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    - Fix the issue that alter enum/set type does not check constraint [#20045](https://github.com/pingcap/tidb/pull/20045)
+    - Fix the panic that tidb-server does not release table locks [#20021](https://github.com/pingcap/tidb/pull/20021)
+    - Fix the behavior of rewrite ScalarFunction IsTure . [#19901](https://github.com/pingcap/tidb/pull/19901)
+    - Support adjusting the concurrency on the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - Set `sync-log` to `true` always [#8636](https://github.com/tikv/tikv/pull/8636)
+    - Fix the bug that TiKV panics when parsing responses with missing reason phrases [#8540](https://github.com/tikv/tikv/pull/8540)
```
Comment (nolouch) [inline releases/release-3.0.19.md:37]: Move to `bug fixes`

File: releases/release-3.0.19.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
++ TiDB
+
+    - Fix the issue that alter enum/set type does not check constraint [#20045](https://github.com/pingcap/tidb/pull/20045)
+    - Fix the panic that tidb-server does not release table locks [#20021](https://github.com/pingcap/tidb/pull/20021)
+    - Fix the behavior of rewrite ScalarFunction IsTure . [#19901](https://github.com/pingcap/tidb/pull/19901)
+    - Support adjusting the concurrency on the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - Set `sync-log` to `true` always [#8636](https://github.com/tikv/tikv/pull/8636)
```
Comment (youjiali1995) [inline releases/release-3.0.19.md:36]: Move to `improvements`.

File: releases/release-3.0.19.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    - Mitigate the impact of failure recovery on QPS [#19764](https://github.com/pingcap/tidb/pull/19764)
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix the `slow_query` error when the `slow-log` file does not exist [#20050](https://github.com/pingcap/tidb/pull/20050)
+    - Add the privilege check for `SHOW STATS_META`, `SHOW STATS_BUCKET` [#19759](https://github.com/pingcap/tidb/pull/19759)
+    - Forbid changing the decimal type to the integer type [#19681](https://github.com/pingcap/tidb/pull/19681)
+
```
Comment (3pointer) [inline releases/release-3.0.19.md:24]:
```suggestion
Tools
- Lightning
    - Fix the issue that break import immediately when detect non-UTF-8 data in strict-mode  [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```

File: releases/release-3.0.19.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+## Bug Fixes
+
++ TiDB
+
+    - Fix the `slow_query` error when the `slow-log` file does not exist [#20050](https://github.com/pingcap/tidb/pull/20050)
+    - Add the privilege check for `SHOW STATS_META`, `SHOW STATS_BUCKET` [#19759](https://github.com/pingcap/tidb/pull/19759)
+    - Forbid changing the decimal type to the integer type [#19681](https://github.com/pingcap/tidb/pull/19681)
+    - Fix the issue that alter enum/set type does not check constraint [#20045](https://github.com/pingcap/tidb/pull/20045)
+    - Fix the panic that tidb-server does not release table locks [#20021](https://github.com/pingcap/tidb/pull/20021)
```
Comment (wjhuang2016) [inline releases/release-3.0.19.md:41]: Fix a bug that tidb-server does not release table locks after a panic.

File: releases/release-3.0.19.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    - Fix the behavior of rewrite ScalarFunction IsTure . [#19901](https://github.com/pingcap/tidb/pull/19901)
```
Comment (wjhuang2016) [inline releases/release-3.0.19.md:42]: Fix a bug that "OR" operation is not handled correctly in where clause

File: releases/release-3.0.19.md (hunk 6 above)
Comment (TomShawn) [inline releases/release-3.0.19.md:41]:
```suggestion
    - Fix the bug that tidb-server does not release table locks after a panic [#20021](https://github.com/pingcap/tidb/pull/20021)
```

File: releases/release-3.0.19.md (hunk 7 above)
Comment (TomShawn) [inline releases/release-3.0.19.md:42]:
```suggestion
    - Fix the bug that the `OR` operation is not handled correctly in the `WHERE` clause [#19901](https://github.com/pingcap/tidb/pull/19901)
```

File: releases/release-3.0.19.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
++ PD
+
+    - Change the import path from `pingcap/pd` to `tikv/pd` [#2779](https://github.com/pingcap/pd/pull/2779)
+    - Change the copyright information from `PingCAP, Inc` to `TiKV Project Authors` [#2777](https://github.com/pingcap/pd/pull/2777)
+
+## Improvements
+
++ TiDB
+
+    - Mitigate the impact of failure recovery on QPS [#19764](https://github.com/pingcap/tidb/pull/19764)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:22]:
```suggestion
    - Mitigate the impact of failure recovery on QPS performance [#19764](https://github.com/pingcap/tidb/pull/19764)
```

File: releases/release-3.0.19.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    - Support adjusting the concurrency on the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:23]:
```suggestion
    - Support adjusting the concurrency of the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
```

File: releases/release-3.0.19.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
++ TiKV
+
+    - Set `sync-log` to `true` always [#8636](https://github.com/tikv/tikv/pull/8636)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:27]:
```suggestion
    - Set `sync-log` to `true` by default [#8636](https://github.com/tikv/tikv/pull/8636)
```

File: releases/release-3.0.19.md [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
++ PD
+
+    - Add an alert rule for restart [#2789](https://github.com/pingcap/pd/pull/2789)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:31]:
```suggestion
    - Add an alert rule for PD restart [#2789](https://github.com/pingcap/pd/pull/2789)
```

File: releases/release-3.0.19.md [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+
+## Bug Fixes
+
++ TiDB
+
+    - Fix the `slow_query` error when the `slow-log` file does not exist [#20050](https://github.com/pingcap/tidb/pull/20050)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:37]:
```suggestion
    - Fix the query error that occurs when the `slow-log` file does not exist [#20050](https://github.com/pingcap/tidb/pull/20050)
```

File: releases/release-3.0.19.md [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+    - Add the privilege check for `SHOW STATS_META`, `SHOW STATS_BUCKET` [#19759](https://github.com/pingcap/tidb/pull/19759)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:38]:
```suggestion
    - Add the privilege check for `SHOW STATS_META` and `SHOW STATS_BUCKET` [#19759](https://github.com/pingcap/tidb/pull/19759)
```

File: releases/release-3.0.19.md (hunk 10 above)
Comment (hunterlxt) [reply releases/release-3.0.19.md:27]: In fact, before this option is true by default, now we have removed this configuration item, so it always appears as true

File: releases/release-3.0.19.md [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+    - Forbid changing the decimal type to the integer type [#19681](https://github.com/pingcap/tidb/pull/19681)
+    - Fix the issue that alter enum/set type does not check constraint [#20045](https://github.com/pingcap/tidb/pull/20045)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:40]: @wjhuang2016 
```suggestion
    - Fix the issue that there is no check for constraint when altering the `ENUM`/`SET` type column [#20045](https://github.com/pingcap/tidb/pull/20045)
```

File: releases/release-3.0.19.md [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+    - Fix the bug that tidb-server does not release table locks after a panic [#20021](https://github.com/pingcap/tidb/pull/20021)
+    - Fix the bug that the `OR` operation is not handled correctly in the `WHERE` clause [#19901](https://github.com/pingcap/tidb/pull/19901)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:42]:
```suggestion
    - Fix the bug that the `OR` operator is not handled correctly in the `WHERE` clause [#19901](https://github.com/pingcap/tidb/pull/19901)
```

File: releases/release-3.0.19.md [hunk 16]
```diff
@@ -0,0 +1,52 @@
...
+
++ TiKV
+
+    - Fix the bug that TiKV panics when parsing responses with missing reason phrases [#8540](https://github.com/tikv/tikv/pull/8540)
+
++ Tools
+
+    + TiDB Lightning
+
+        - Fix the issue that break import immediately when detect non-UTF-8 data in strict-mode  [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:52]:
```suggestion
        - Fix the issue that the TiDB Lightning process does not exit in time when encountering illegal UTF characters in the strict mode [#378](https://github.com/pingcap/tidb-lightning/pull/378)
```

File: releases/release-3.0.19.md (hunk 10 above)
Comment (TomShawn) [reply releases/release-3.0.19.md:27]: Thanks! So it is safe to say "set it to true permanently", right?

File: releases/release-3.0.19.md [hunk 17]
```diff
@@ -0,0 +1,52 @@
...
+## Improvements
+
++ TiDB
+
+    - Mitigate the impact of failure recovery on QPS performance [#19764](https://github.com/pingcap/tidb/pull/19764)
+    - Support adjusting the concurrency of the `union` operator [#19885](https://github.com/pingcap/tidb/pull/19885)
+
++ TiKV
+
+    - Set `sync-log` to `true` as an nonadjustable value [#8636](https://github.com/tikv/tikv/pull/8636)
```
Comment (TomShawn) [inline releases/release-3.0.19.md:27]:
```suggestion
    - Set `sync-log` to `true` as a nonadjustable value [#8636](https://github.com/tikv/tikv/pull/8636)
```

**Comment by TomShawn** [comment]
LGTM

**Review by wjhuang2016 — COMMENTED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]

**Pull request merged**